.env.example

```
@@ -10,4 +10,5 @@ EXPO_PUBLIC_GOOGLE_IOS_CLIENT_ID=
 EXPO_PUBLIC_GOOGLE_ANDROID_CLIENT_ID=
 GOOGLE_REFRESH_TOKEN=
 GOOGLE_CLIENT_SECRET=
-GOOGLE_CLIENT_ID=
+GOOGLE_CLIENT_ID= 
```

dotenv-linter (4.0.0)

[warning] [SpaceCharacter] The line has spaces around equal sign

(SpaceCharacter)